api/disk.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+)
+
+func (api *API) ResizeDisk(instanceID, extraDiskSize int) (map[string]interface{}, error) {
+	data := make(map[string]interface{})
+	failed := make(map[string]interface{})
+	id := strconv.Itoa(instanceID)
+	log.Printf("[DEBUG] go-api::disk::resize instance ID: %s", id)
+	path := fmt.Sprintf("api/instances/%s/disk?extra_disk_size=%d", id, extraDiskSize)
```

Comment:
We should send `extra_disk_size` in the body, not as parameter (I know it might work with query params, but I think that is a bit quirky)